src/app/@ansyn/status-bar/components/status-bar/status-bar.component.ts

```diff
@@ -148,7 +203,36 @@ export class StatusBarComponent implements OnInit {
 			this.time = _time;
 		});
 
+		this.preFilter$.subscribe(_preFilter => {
+			this._selectedFilters = _preFilter;
+			if (Boolean(this._selectedFilters)) {
+				this.updateInputDataFilterMenu();
+			}
+		});
+	}
+
+	toggleDataInputFilterIcon() {
+		this.dataInputFilterIcon = !this.dataInputFilterIcon;
+		if (this.dataInputFilterIcon) {
+			this._oldSelectedFilters = this._selectedFilters;
+		}
+	}
 
+	updateInputDataFilterMenu(): void {
+		this.dataInputFiltersItems.forEach((dataInputItem) => {
```

Comment:
Add a comment explaining why you are iterating the UI filters instead of the case data input filters.

Reply:
Done